src/components/modal/ModalHeader.tsx

```diff
@@ -6,15 +6,15 @@ const MODAL_HEADER_TITLE: { [name: string]: string } = {
   app: 'Create a new app',
 };
 
-export const ModalHeader = ({ modalType, close }: { modalType: string; close: () => void }): React.ReactElement => {
+export const ModalHeader = ({ modalType, close }: { modalType?: string; close: () => void }): React.ReactElement => {
```

Comment:
`modalType` prop type can be strict. use MODAL_TYPES const defined in Modal.tsx, maybe now store constants in a separate constants file.